### PR TITLE
ENDOC-201 Improve the bundle publish tutorial.

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

File: vuepress/docs/next/tutorials/ecr/publish-project-bundle.md [hunk 1]
```diff
@@ -154,3 +154,5 @@ kubectl apply -f example-bundle.yaml -n entando
 3. Find your bundle and select `Install`
 
 At this point the Entando platform will download and install the Docker image for your microservice and install the micro frontends into the Entando application. You can add those micro frontend widgets to the page(s) of your choice.
+
+You can use an Entando-provided page or follow [this tutorial](../cms/page-management.md) to create your own page and/or page template. Please note that an Entando blueprint-generated application expects a user to be authenticated so you'll need to include the `keycloak_auth` UX fragment in your page template.
```
Comment (joewhite101) [inline vuepress/docs/next/tutorials/ecr/publish-project-bundle.md:158]: `Entando-provided` Is that hyphen right? Might be but not sure. My hyphen game is not strong but looks odd to me ...

Comment (joewhite101) [reply vuepress/docs/next/tutorials/ecr/publish-project-bundle.md:158]: Same with `blueprint-generated`

**Review by joewhite101 — APPROVED** [review]
LGTM.

File: vuepress/docs/next/tutorials/ecr/publish-project-bundle.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/publish-project-bundle.md:158]: Yeah, it's bugging me now. I'll drop the hyphens.

**Pull request merged**